packages/graphql/src/codegen/generate.ts

```diff
@@ -40,28 +38,11 @@ export async function generate({
 }: GenerateOptions): Promise<void> {
   const config: Partial<Types.Config> = {
     schema: "./**/*.graphql",
+    documents: "./**/*.graphql",
+    ignoreNoDocuments: true,
```

Comment:
Nice, can you add a little comment here too explaining ignoreNoDocuments? Is there a relevant docs link we can include?